Makefile.inc

```diff
@@ -31,7 +32,8 @@ endif
 
 # g++ flags used in all g++ calls.
 CPP_COMMON := -std=c++17 -fPIE -pthread -Wall ${c_opts} $(GCOVR_FLAGS)
-CPP_LIBS := -L/usr/local/lib -lyaml-cpp
+#&&&CPP_LIBS := -L/usr/local/lib -lyaml-cpp
+CPP_LIBS += -L/usr/local/lib $(shell pkg-config yaml-cpp --libs) $(shell pkg-config spdlog --libs) -ldl
```

Comment:
Please pass $silence flag for all pkg-config calls as in ts_m1m3support:

- in ifndef VERBOSE, define it:
```
ifndef VERBOSE
  co := @
  silence := --silence-errors
endif
```

use it:

```
PKG_CPPFLAGS := $(shell pkg-config yaml-cpp --cflags $(silence)) $(shell pkg-config spdlog --cflags $(silence))
```

etc. The benefit is that if you have a locally available spdlog in include path (no pkg-config needed, no .pc files), you will not get an error running pkg-config. Allows you to provide packages when updates aren't available - that happened in the past.